app/src/main/java/ch/epfl/sdp/blindly/profile_setup/ProfileGender.kt

```diff
@@ -37,30 +39,29 @@ class ProfileGender : AppCompatActivity() {
             }
             //more option is checked
             radioButtonMore.isChecked -> {
-                val intentMore = Intent(this, ProfileGenderMore::class.java)
-                val genre = "More"
-                bundleExtrasAndStartActivity(intentMore, username, birthday, genre)
+                nextIntent = Intent(this, ProfileGenderMore::class.java)
+                gender = "More"
```

Comment:
The genders (More, Woman and Man) should be in private const vals imo.

Reply:
I don't know if it's better since they're used in bundleExtrasAndStartActivity(), this way we don't need to pass as argument (gender) each time
Or did you mean creating a private const val MAN = "Man" ?

Reply:
Yeah that's what I meant